src/components/signin/Mnemonics.js

```diff
@@ -60,7 +61,7 @@ const Mnemonics = ({ screenProps, navigation, styles }) => {
 
     const showError = () => {
       log.error('Incorrect pass phrase - wallet recover failed', '', null, {
-        mnemonics,
+        mnemonic: mnemonics,
```

Comment:
what if we'll miss up in another place ?
should add all types possIble. please adjust default values of `REACT_APP_LOG_SECURE_KEYS` to:

`mnemonic*,pass*,privateKey,mobile,email,seed`

or (if won't work):

`mnemonic,mnemonics,password,pass,privateKey,mobile,email,seed`

Reply:
I checked and there are no 'mnemonics' logged elsewhere. Only 'mnemonic' will appear in the console.

Reply:
Okay but what if we'll forgot that only 'mnemonic' is censored in the future and we'll log 'mnemonics' ?
We have to foresee this. So please try to adjust secure keys